misc/ls-extensions/modules/json-to-record-converter/src/main/java/io/ballerina/converters/JsonToRecordConverterService.java

```diff
@@ -49,26 +54,30 @@ public Class<?> getRemoteInterface() {
     @JsonRequest
     public CompletableFuture<JsonToRecordResponse> convert(JsonToRecordRequest request) {
         return CompletableFuture.supplyAsync(() -> {
-            JsonToRecordResponse response;
-            try {
-                String jsonString = request.getJsonString();
-                String recordName = request.getRecordName();
-                boolean isRecordTypeDesc = request.getIsRecordTypeDesc();
-                boolean isClosed = request.getIsClosed();
+            JsonToRecordResponse response = new JsonToRecordResponse();
+
+            String jsonString = request.getJsonString();
+            String recordName = request.getRecordName();
+            boolean isRecordTypeDesc = request.getIsRecordTypeDesc();
+            boolean isClosed = request.getIsClosed();
 
-                ObjectMapper objectMapper = new ObjectMapper();
-                JsonNode inputJson = objectMapper.readTree(jsonString);
-                if (inputJson.has("$schema")) {
-                    response = JsonToRecordConverter.convert(jsonString, recordName, isRecordTypeDesc, isClosed);
+            try {
+                JsonElement parsedJson = JsonParser.parseString(jsonString);
+                if (parsedJson.isJsonObject() && parsedJson.getAsJsonObject().has("$schema")) {
+                    try {
+                        response.setCodeBlock(JsonToRecordConverter.convert(jsonString, recordName, isRecordTypeDesc,
+                                isClosed).getCodeBlock());
+                    } catch (IOException | JsonToRecordConverterException | FormatterException |
```

Comment:
Is it possible to use a generic `Exception`  instead of using a separate exceptions?

Reply:
Using `Exception` instead of its subclasses cause build failure. The performance check something.

Reply:
@AzeemMuzammil it should be spotbugs. because Catching `Exception`s are generally not recommended, since it will catch all the runtime exceptions. But in case you really need to do that, we can ignore the spotbug warning and catch `Exception`